longest-string.js

```diff
@@ -0,0 +1,14 @@
+var pry = require('pryjs')
+
+function longestString(words) {
+  var word = "c";
```

Comment:
What would happen if the `words` array was empty? we would probably want to return `null` rather than "c". Some error checking would help here, maybe you could initialize `var word = words[0]` then start from the second word in the loop